localisation/00_wc_economy.csv

```diff
@@ -0,0 +1,19 @@
+#CODE;ENGLISH;FRENCH;GERMAN;;SPANISH;;;;;;;;;x
+borrow_money;Borrow 300�;;Leiht Euch 300�;;;;;;;;;;x;
+borrow_money_desc;You can borrow 300� from some rich ruler or moneylenders.;;Ihr k�nnt Euch 300� von einem reichen Herrscher oder Geldverleiher leihen.;;;;;;;;;;x;
+repay_money;Repay 360� Loan;;Zahlt 360� Darlehen zur�ck;;;;;;;;;;;x
+repay_money_desc;You repay the gold you owe.;;Ihr Zahlt das Darlehen zur�ck, das Ihr schuldig seid.;;;;;;;;;;;x
```

Comment:
How about this instead of double "you"?

```suggestion
repay_money_desc;[Root.GetBestName] repay the gold [Root.GetSheHe] owe.;;Ihr Zahlt das Darlehen zur�ck, das Ihr schuldig seid.;;;;;;;;;;;x
```

Reply:
@MemeWizard counteroffer, how about?
```suggestion
repay_money_desc;Repay the gold you owe.;;Ihr Zahlt das Darlehen zur�ck, das Ihr schuldig seid.;;;;;;;;;;;x
```

Reply:
Sounds ok too

Reply:
Done in f9b35ad6880bb7d01e2c57e447108b78ac6a1a2a.